tests/introspection/Mac/MacApiCtorInitTest.cs

```diff
@@ -288,6 +288,11 @@ protected override void Dispose (NSObject obj, Type type)
 					goto default;
 				do_not_dispose.Add (obj);
 				break;
+			case "AppKit.NSTextInputContext":
+			case "MonoMac.AppKit.NSTextInputContext":
+				if (Mac.CheckSystemVersion (10, 13))
```

Comment:
Attempting to dispose NSTextInputContext on the 10.13.2 beta causes the test to crash, it works fine on 10.13.1 stable.  The CheckSystemVersion method does not currently allow minor builds, only checking for major (do we want to change that?)

Reply:
1. Yes, we should be able to check the 3 version digit on macOS since it's something we support in the availability attribute (so our tests should be able to match).

2. Since this is a beta I rather not commit this right now. It might be fixed (please file a rdar) and it's too easy to forget removing the exclusion (even more since the current API does not show the revision, nor does it have a comment about which beta is affected).

Now it's not a big deal for this specific test as it does not block further checks - but it would be bad for some other intro tests.